.speakeasy/gen.lock

```diff
@@ -36,6 +36,7 @@ generatedFiles:
   - examples/data-sources/konnect_api_product_specification/data-source.tf
   - examples/data-sources/konnect_api_product_version/data-source.tf
   - examples/data-sources/konnect_application_auth_strategy/data-source.tf
+  - examples/data-sources/konnect_audit_log/data-source.tf
```

Comment:
@mheap I'm not sure if this should be there, since this file does not exist - could you please suggest if I should change something?

Reply:
The file is generated then removed. There is no harm in leaving this line in place